scss/_patterns_notifications.scss

```diff
@@ -232,7 +252,7 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
 @mixin vf-notifications-negative {
```

Comment:
Please update the "caution" and "information" notifications as well to use theme colours.